proto/node/v1/node.proto

```diff
@@ -142,6 +147,22 @@ message BridgeUpgradeContract {
   string new_contract = 3;
 }
 
+message WormchainStoreCode {
+  // payload is the hex string of the sha3 256 hash of the wasm binary being uploaded
+  string wasm_hash = 1;
```

Comment:
Protobufs have a `bytes` type so why not use the bytes directly rather than the hex?

Reply:
My thought here was it'd be easier for a user to pass in the string hex than binary when constructing a prototxt message.